nuget/lib/dependabot/nuget/nuget_client.rb

```diff
@@ -68,7 +68,11 @@ def self.get_package_versions(dependency_name, repository_details)
             # inlined entries
             items.each do |item|
               catalog_entry = item["catalogEntry"]
-              if catalog_entry["listed"] == true
+
+              # a package is considered listed if the `listed` property is either `true` or missing
+              listed_property = catalog_entry["listed"]
```

Comment:
Ah, good catch!

Reply:
This part may or may not have taken me 2 hours to figure out.